js-miniapp-sample/src/pages/landing.js

```diff
@@ -13,6 +13,16 @@ type LandingProps = {
   getHostInfo: Function,
 };
 
+window.addEventListener('miniAppWebviewClosed', function (e) {
+  let message = e.detail;
+  console.log(message);
+});
+
+window.addEventListener('miniAppOnPause', function (e) {
```

Comment:
I think these should be `miniappwebviewclosed` and `miniapppause` instead. It's not as easy to read, but it's the standard format used by JavaScript events (see https://developer.mozilla.org/en-US/docs/Web/Events)